src/one_dragon_qt/view/installer_interface.py

```
@@ -1,919 +1,278 @@
-    def update_log_display(self):
-        """更新日志显示内容"""
-        if not hasattr(self, 'log_receiver') or not hasattr(self, 'log_display_label'):
+            self._installing = False
+            self._installing_idx = -1
+            self.log_display.stop()
+            self.install_btn.setText(gt('一键安装'))
+            self.install_btn.setDisabled(False)
+            self.step_stack.setCurrentIndex(1)  # 全部安装完成，切到成功页
+
+    def on_install_progress(self, progress: float, message: str) -> None:
+        """安装进度回调：更新阶段条与状态文字。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)